File: CpuGpu.java
import java.io.BufferedReader;

public class CpuGpu {

  public static void main(String[] args) throws I0Exception {
    try {
      ProcessBuilder launcher = new ProcessBuilder();
      Map < String,String > environment = launcher.environment();
      launcher.redirectErrorStream(true);
      //destinationOfWindowsPowerShellExe must be replaced with the path of the Windows PowerShell Exe
      launcher.directory(new File("destinationOfWindowsPowerShellExe"));
      launcher.command("powershell.exe", "destinationOfPowershellFile");
      Process p = launcher.start() : // And launch a new process
      BufferedReader stdInput = new BufferedReader(new InputStreamReader(p.getInputStream()));
      String line;
      while ((line = stdInput.readline()) != null) {
        System.out.println(line);
      }
    }
    catch(Exception e) {
      e.printStackTrace;
    }
  }
}

File: CpuGpu.py
import subprocess

def main():
    try:
        # Start a new subprocess to run a PowerShell command or script.
        # 'powershell.exe' is the command to launch PowerShell.
        # 'destinationOfPowershellFile' should be replaced with the actual path to your PowerShell script or command.
        launcher = subprocess.Popen(['powershell.exe', 'destinationOfPowershellFile'], stdout=subprocess.PIPE, stderr=subprocess.STDOUT, shell=True)
        
        # Capture the output (stdout) of the PowerShell command.
        stdout, _ = launcher.communicate()
        
        # Split and print each line of the captured output.
        for line in stdout.decode('utf-8').splitlines():
            print(line)
    
    except Exception as e:
        # Handle any exceptions that may occur during the subprocess execution.
        print(e)

if __name__ == '__main__':
    # This block of code will be executed when the script is run directly (not imported as a module).
    main()
